network/dns-cloud/how-to/manage-nameservers-internal-domain.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: How to manage nameservers for an internal domain name
+  description: This page explains how to manage nameservers for an internal domain name with Scaleway Domains & DNS.
+content:
+  h1: How to manage nameservers for an internal domain name
+  paragraph: This page explains how to manage nameservers for an internal domain name with Scaleway Domains & DNS.
+dates:
+  validation: 2022-10-12
+  posted: 2022-10-12
+---
+
+Scaleway Domains & DNS provides a managed DNS solution by providing default nameservers for each domain name. If you want to use your own DNS servers, you can modify the nameservers associated with your domain name.
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+- You have a domain name registered with Scaleway Domains & DNS
+
+</Message>
+
+1. Click **Domains & DNS** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. 
+
+ The Domain & DNS dashboard displays. 
+ 
+2. Click the domain name you want to configure.
+
+3. Click **Edit** in the **Nameservers** section of the domain overview page. 
+
+4. Edit the nameserver information. For each nameserver of your domain you can provide:
+    * A fully qualified domain name (FQDN)
```

Comment:
It only says "nameserver" now, should we replace this by "A nameserver"?

Reply:
looks ok to me ?